packaging/rpm/rear.spec

```diff
@@ -18,8 +18,7 @@ License: GPL-3.0
 Group: Applications/File
 URL: http://relax-and-recover.org/
 
-# as GitHub stopped with download section we need to go back to Sourceforge for downloads
-Source: https://sourceforge.net/projects/rear/files/rear/%{version}/rear-%{version}.tar.gz
+Source: https://github.com/rear/rear/archive/%{version}.tar.gz#/rear-%{version}.tar.gz
```

Comment:
In which circumstance or context does this line actually take effect if `make dist` always changes that to just the dist archive file name?

Reply:
Only when you want to build the RPM manually and need to know where to obtain the sources from. In the future, we might also avoid spec file rewriting in `make dist` if `OFFICIAL` is set and thus the RPM is goping to be built from unmodified sources.